src/main/java/io/anserini/document/Status.java

```diff
@@ -0,0 +1,213 @@
+      this.followers_count = followers_count;
+      this.friends_count = friends_count;
+      this.statuses_count = statuses_count;
+      this.screen_name = screen_name;
+    }
+
+    @JsonGetter("screen_name")
+    public String screen_name() {
+      return screen_name;
+    }
+
+    @JsonGetter("followers_count")
+    public int followers_count() {
+      return followers_count;
+    }
+
+    @JsonGetter("friends_count")
+    public int friends_count() {
+      return friends_count;
+    }
+
+    @JsonGetter("statuses_count")
+    public int statuses_count() {
+      return statuses_count;
+    }
+
+    @JsonGetter("name")
+    public String name() {
+      return name;
+    }
+
+    @JsonGetter("profile_image_url")
+    public String profile_image_url() {
+      return profile_image_url;
+    }
+
+    @JsonGetter("id")
+    public long id() {
+      return id;
+    }
+  }
+
+  @JsonCreator
+  public Status(
+    @JsonProperty(value = "created_at", required = true) String created_at,
+    @JsonProperty(value = "id_str", required = true) String id_str,
+    @JsonProperty(value = "text", required = true) String text,
+    @JsonProperty(value = "user", required = true) User user) {
+    this.created_at = created_at;
+    this.id_str = id_str;
+    this.text = text;
+    this.user = user;
+  }
+
+  @JsonGetter("id_str")
+  public String id_str() {
+    return id_str;
+  }
+
+  @JsonGetter("text")
+  public String text() {
+    return text;
+  }
+
+  @JsonGetter("user")
+  public User user() {
+    return user;
+  }
+
+  @JsonGetter("created_at")
+  public String created_at() {
+    return created_at;
+  }
+
+  @JsonGetter("retweet_count")
+  public long retweet_count() {
+    return retweet_count;
+  }
+
+  @JsonSetter("retweet_count")
+  public void set_retweet_count_internal(JsonNode retweet_count_internal) {
+    if (retweet_count_internal != null) {
+      if (retweet_count_internal.isTextual()) {
+        // retweet_count might say "100+"
+        // TODO: This is ugly, come back and fix later.
+        retweet_count = Long.parseLong(retweet_count_internal.asText().replace("+", ""));
+      } else if (retweet_count_internal.isNumber()) {
+        retweet_count = retweet_count_internal.asLong();
+      }
+    }
+  }
+
+  @JsonGetter("in_reply_to_status_id")
+  public OptionalLong in_reply_to_status_id() {
+    return in_reply_to_status_id;
+  }
+
+  @JsonGetter("in_reply_to_user_id")
+  public OptionalLong in_reply_to_user_id() {
+    return in_reply_to_user_id;
+  }
+
+  @JsonGetter("retweeted_status")
+  public Optional<RetweetedStatus> retweeted_status() {
+    return retweeted_status;
+  }
+
+  @JsonGetter("lang")
+  public Optional<String> lang() {
+    return lang;
+  }
+
+  @JsonGetter("delete")
+  public Optional<Delete> delete() {
+    return delete;
+  }
+
+  @JsonGetter("coordinates")
+  public Optional<Coordinates> coordinates() { return coordinates; }
+
+  //TODO: to be deleted before merging
```

Comment:
What to do with this?

Reply:
An overloaded getter method for Jackson deserialization